src/main/java/cn/nukkit/entity/passive/EntityNpc.java

```diff
@@ -0,0 +1,57 @@
+package cn.nukkit.entity.passive;
+
+import cn.nukkit.entity.EntityInteractable;
+import cn.nukkit.entity.EntityLiving;
+import cn.nukkit.level.format.FullChunk;
+import cn.nukkit.nbt.tag.CompoundTag;
+
+/**
+ * @author good777LUCKY
+ */
+public class EntityNpc extends EntityLiving implements EntityNPC, EntityInteractable {
```

Comment:
Source code class name conflict with `EntityNPC` in Windows environment, please change to a different name.

Reply:
I changed npc class name to `EntityNPCEntity`.